cats/templates/cats/cats_list.html

```diff
@@ -0,0 +1,68 @@
+{% extends 'base.html' %}
+{% load crispy_forms_tags %}
+{% block title %}Cats list{% endblock %}
+{% block content %}
+    <h1>{{ species.name }}</h1>
+    {% if not species %}
+        <div style="text-align: center; background-color:orange; margin: auto">
+            No such species id
+        </div>
+    {% elif cats_list == None %}
+        <div style="text-align: center; background-color:orange; margin: auto">
+            Search cats available in given dates
+        </div>
+        <div style="width: 200px; text-align: center; margin: auto">
+            <form method="get">
+                {{ search_form }}
+                    <button type="submit">Search</button>
+            </form>
+        </div>
+    {% else %}
+        <hr class="rounded">
+        <table class="table table-hover">
+            <thead>
+            <td>Name</td>
+            <td>Breed</td>
+            <td>Last rental</td>
+            <td>Last return</td>
+            </thead>
```

Comment:
You can work on indentation in this file, for example here
```suggestion
            <thead>
                <td>Name</td>
                <td>Breed</td>
                <td>Last rental</td>
                <td>Last return</td>
            </thead>
```